File: csharp/two-fer/TwoFer.cs
namespace TwoFer
{
	public static class TwoFer
	{
		public static string Name(string input = null)
		{
			var output = string.IsNullOrEmpty(input) ? "you" : input;
			return $"One for {output}, one for me.";
		}
	}
}

File: java/two-fer/src/main/java/Twofer.java
class Twofer {

    // Creates Two-fer string
    String twofer(String name) {
        final var mainString = "One for , one for me.";
        final var offset = mainString.indexOf(',');

        if (name == null) {
            name = "you";
        }

        var result = new StringBuilder(mainString).insert(offset, name);

        return result.toString();
    }
}
